app/services/geocode_service.rb

```diff
@@ -0,0 +1,28 @@
+class GeocodeService
```

Comment:
May want to think about renaming this `google service`. You should be able to use this service with all google api endpoints not just the geocoding endpoint. Hint: you will have to use another google api endpoint for the assessment. Don't create a whole new service reuse the same one.